setup.py

```diff
@@ -0,0 +1,2 @@
+import setuptools
+setuptools.setup()
```

Comment:
We had talked (albeit briefly) about holding off on _packaging_ this repo as a self-contained Python package for installation. I'm not entirely convinced it's an overhead we want to take on at this point, but I could certainly be convinced. 

If we do want to do this up front, then I think we should include more documentation about how to package it (what commands to run). It could also be useful to know what we get as general users/developers by building and installing the package. Is it required to build and install the package to run the code contained here-in, or is installing a sufficient conda environment (for example, maybe it's a virtualenv with pip) enough to do development and testing?

Reply:
I don't see an overhead. 
A python module must be packaged.

There is never enough documentation, but I agree we need to add a little more basic commands such as:
```
pip install .
```

Or however more verbose that is appropriate.

Like I said, this PR is setting up basic infrastructure I would expect from all the workflow work.

Reply:
Added simple instructions for installation.